notebooks/book2/29/kf_parallel.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>I suggested something like this,</p><p><code>LINE_WIDTH1=0.7 if is_latexify_enabled() else None</code></p><p>Here None will be considerd as default line width</p><p> </p><p>Try to understand the reason behind my suggestions :)</p><p>I have suggested this so we can decrease code complexity (in terms of readibily), but it does not seems to be, right? We don't want to put <code>is_latexify_enabled()</code> stuffs in between of code. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
@karm-patel Bhaiya, sorry for misinterpreting the `LINE_WIDTH`. I have changed the notebook as per your suggestions. Let me know if I have missed out on something.